pipelines/disturbance/stages.py

```diff
@@ -19,17 +19,19 @@ def setup_compute(
     expected_groups: Optional[ExpectedGroupsType],
     contextual_column_name: Optional[str] = None,
 ) -> Tuple:
-    """Setup the arguments for the xrarray reduce on dist alerts"""
-    dist_alerts, country, region, subregion, contextual_layer = datasets
+    """Setup the arguments for the xarray reduce on dist alerts"""
+    dist_alerts, country, region, subregion, pixel_area, contextual_layer = datasets
 
-    mask = dist_alerts.confidence
+    mask = pixel_area.band_data
```

Comment:
minor naming, but I think when calculating area, this is functioning less as a "mask" than an aggregate layer.